wyklady/wyklad2.txt

```diff
@@ -0,0 +1,156 @@
+deck_id: 3892092678
+deck_name: Sieci komputerowe MIMUW
+file_id: sik-wyklad-2
+
+
+1
+Czym są programy sieciowe?
+To programy uruchamiane na systemach końcowych komunikujące się ze sobą za pośrednictwem sieci.
+
+2
+Podaj główne typy architektur aplikacji sieciowych.
+Architektura klient-serwer, architektura peer-to-peer (P2P), architektura hybrydowa (mieszanka obu).
```

Comment:
Usunąłbym wielokrotne powtórzenie słowa "architektura" w odpowiedzi (nic nie wnosi, a wydłuża fiszkę), z racji że z pytania wynika że mowa o architekturach, to można tego słowa się pozbyć w odpowiedzi.